pkg/webhook/storageclass.go

```diff
@@ -122,38 +69,35 @@ func mutateStorageClass(ar v1.AdmissionReview) *v1.AdmissionResponse {
 		Result:  &metav1.Status{},
 	}
 
-	if !(ar.Request.Operation == v1.Update || ar.Request.Operation == v1.Create) {
+	if ar.Request.Operation == v1.Update {
+		return rejectV1AdmissionResponse(fmt.Errorf("resource %v is immutable", &StorageClassV1GVR))
+	} else if ar.Request.Operation != v1.Create {
```

Comment:
Let API server handle any mutation rule. We should simply return success response to anything other than Create.

Reply:
done